src/pages/Profile/Mypage/index.tsx

```diff
@@ -68,18 +72,18 @@ const Mypage: React.FC<MypageProps> = ({
       <div className="_ITEM_">
         <Box>
           <AchievementBadge
-            id={1}
-            title="5급 조타기"
-            description="길잡 신규가입 시 달성"
+            id={exp_achievement.list[0].id}
+            title={exp_achievement.list[0].title}
+            description={exp_achievement.list[0].description}
```

Comment:
여기 타입 정의가 안 된 객체를 사용하고 있어서 에러가 발생할 확률이 있기 때문에 optional chaining 해야 할 것 같아요 🤔 
```suggestion
            id={exp_achievement?.list[0]?.id}
            title={exp_achievement?.list[0]?.title}
            description={exp_achievement?.list[0]?.description}
```